Source/DafnyLanguageServer.Test/Various/ConcurrentInteractionsTest.cs

```diff
@@ -73,6 +84,70 @@ public class ConcurrentInteractionsTest : DafnyLanguageServerTestBase {
       Assert.AreEqual(1, diagnostics.Length);
     }
 
+    [TestMethod, Timeout(MaxTestExecutionTimeMs)]
+    public async Task VerificationErrorDetectedAfterCanceledSave() {
+      // Create a document that'll be slightly slow to verify
+      var source = @"
+method Multiply(x: int, y: int) returns (product: int)
+  requires y >= 0 && x >= 0
+  decreases y
+  ensures product == x * y && product >= 0
+{
+  if y == 0 {
+    product := 0;
+  } else {
+    var step := Multiply(x, y - 1);
+    product := x + step;
+  }
+}".TrimStart();
+      var failSource = @"method Contradiction() { assert false; }";
+      await SetUp(new Dictionary<string, string>() {
+        { $"{DocumentOptions.Section}:{nameof(DocumentOptions.Verify)}", nameof(AutoVerification.OnSave) }
+      });
+      var documentItem = CreateTestDocument(source);
+      await client.OpenDocumentAndWaitAsync(documentItem, CancellationTokenWithHighTimeout);
+
+      DidChangeTextDocumentParams makeChange(int? version, Range range, string text) {
+        return new DidChangeTextDocumentParams {
+          TextDocument = new OptionalVersionedTextDocumentIdentifier {
+            Uri = documentItem.Uri,
+            Version = version
+          },
+          ContentChanges = new[] {
+            new TextDocumentContentChangeEvent {
+              Range = range,
+              Text = text
+            }
+          }
+        };
+      }
+
+      // Add a space in the document
+      var change1 = makeChange(documentItem.Version + 1, new Range((0, 6), (0, 7)), " ");
+      client.DidChangeTextDocument(change1);
+      // Save and don't wait, so the next save will interrupt and cancel it
```

Comment:
Could you change `it` into `verification` ? That's what's being interrupted right, not the saving? Same for line 135

Reply:
Good point. Updated.